src/fs/protocol/private/mmpt.test.ts

```diff
@@ -1,8 +1,9 @@
 import crypto from "crypto"
-import Ipfs, { IPFS } from "ipfs"
+import { IPFS } from "ipfs-core"
```

Comment:
Is this a size thing? (full IPFS includes more code than we use, etc?)

Reply:
Yep, exactly. It's just much less dependencies.
I think the difference is that ipfs also contains the CLI and an http-server, which ipfs-core does not.